app/assets/stylesheets/custom.scss

```diff
@@ -48,16 +48,32 @@ table a, a:hover{
   font-weight: 400;
 }
 
+.navbar-brand{
+  font-family: "SF Sports Night";
+}
+
 .page-header {
   border: 0;
   color: #fff;
-  float: right;
   font-weight: 500;
+  margin-top: 10%;
+  text-shadow: 1px 1px 2px black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.